geoapi/tasks/external_data.py

```diff
@@ -167,7 +167,7 @@ def _handle_point_cloud_conversion_error(pointCloudId, userId, files, error_desc
                                     f"Processing failed for point cloud ({pointCloudId})!")
 
 
-@app.task(rate_limit="1/s")
+@app.task(queue='heavy')
```

Comment:
Just a suggestion - Should we add a logger to let us know which queue a task is using? Probably not needed just curious if that would help us down the line.

Reply:
the task itself gets logged but not which queue it was in. we could have a base celery task that our tasks use and could log that info. good idea when we make improvements to this.